include/openssl/bio.h.in

```diff
@@ -188,6 +188,11 @@ extern "C" {
 # define BIO_DGRAM_CAP_PROVIDES_SRC_ADDR    (1U << 2)
 # define BIO_DGRAM_CAP_PROVIDES_DST_ADDR    (1U << 3)
 
+/*
+ * internal BIO:
+ * # define BIO_CTRL_SET_KTLS_TX_ZEROCOPY_SENDFILE 85
```

Comment:
Please note the value overlaps with BIO_CTRL_DGRAM_GET_EFFECTIVE_CAPS. Please change it. Also the comment should be placed above the BIO_DGRAM_CAP.... defines.

Also the comment on line 233 needs to be updated to reserve the internal BIO_FLAGS value added.